README.md

```diff
@@ -12,15 +14,39 @@ fastlane add_plugin run_tests_firebase_testlab
 
 ## About run_tests_firebase_testlab
 
+Authenticates with Google Cloud.
 Runs Android tests in Firebase testlab.
-
-**Note to author:** Add a more detailed description about this plugin here. If your plugin contains multiple actions, make sure to mention them here.
+Fetches the results to a local directory.
+Deletes the results from firebase bucket if wanted.
 
 ## Example
 
-Check out the [example `Fastfile`](fastlane/Fastfile) to see how to use this plugin. Try it by cloning the repo, running `fastlane install_plugins` and `bundle exec fastlane test`.
+Check out the [example `Fastfile`](fastlane/Fastfile) to see how to use this plugin.
+
+### Simple usage
+
+```
+run_tests_firebase_testlab(
+    project_id: "your-firebase-project-id",
+    model: "Nexus6P",
+    version: "27")
+```
 
-**Note to author:** Please set up a sample project to make it easy for users to explore what your plugin does. Provide everything that is necessary to try out the plugin in this project (including a sample Xcode/Android project if necessary)
+### Parameters
+
+- `project_id`: Your Firebase project id. This parameter is necessary.
+- `model`: The device model name to run the tests. This parameter is necessary.
```

Comment:
I'm not sure about this sentence, maybe something like "The device's model on which the tests will be run."